addon/components/o-s-s/private/base-modal.ts

```diff
@@ -16,7 +16,7 @@ export default class BaseModal<T extends BaseModalArgs> extends Component<T> {
   private prevBodyPadding: string | null = null;
 
   @action
-  destroy(): void {
+  onDestroy(): void {
```

Comment:
naming change here because I was sick of seeing :
![Screenshot 2024-05-03 at 17 37 38](https://github.com/upfluence/oss-components/assets/5032005/61c5ced9-0714-4ef5-80da-0ddde051f44b)
and
![Screenshot 2024-05-03 at 17 37 32](https://github.com/upfluence/oss-components/assets/5032005/5f67a5e8-e435-48b5-8b77-8acd113516a4)